packaging/Dockerfile.centos8-nmstate-dev

```diff
@@ -17,6 +17,7 @@ RUN dnf copr enable nmstate/ovs-el8 -y && \
 RUN dnf -y install --setopt=install_weak_deps=False \
                    NetworkManager \
                    NetworkManager-ovs \
+                   NetworkManager-config-server \
```

Comment:
IMHO it should be a RPM dependency if we want to enforce this. If it is just a workaround, IMHO the bug report should be mentioned. Eventually, https://nmstate.atlassian.net/browse/NMSTATE-17 should make it not needed anymore AFAIU.